### PR TITLE
Added ProcessTime class

### PR DESCRIPTION
**Pull request by rafamanzo**
Signed off by: Diego Araújo <diegoamc90@gmail.com>

### CODE REVIEW

File: kalibro_client/processor/process_time.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from kalibro_client.base import attributes_class_constructor, \
+    entity_name_decorator
+from kalibro_client.processor.base import Base
+
+@entity_name_decorator
+class ProcessTime(attributes_class_constructor('ProcessTimeAttr',
+                                               (('state', None),
```
Comment (pedroscocco) [inline kalibro_client/processor/process_time.py]: ProcessTime validates state presence. Should we keep a default value?

File: tests/test_processor.py [hunk 2]
```diff
@@ -38,6 +41,21 @@ def test_repositories(self):
...
+        self.subject = ProcessTimeFactory.build()
+
+    def test_properties_getters(self):
+        assert_true(hasattr(self.subject, 'time'))
+
+    @not_raises((AttributeError, ValueError))
+    def test_properties_setters(self):
+        self.subject.time = "1"
+
+    def test_time_setter_conversion_to_string(self):
```
Comment (pedroscocco) [inline tests/test_processor.py]: Did you mean `conversion_to_integer`?

**Comment by danielkza** [comment]
I've rebased to master and removed the id in the factory.

**Comment by danielkza** [comment]
Had to rebase again :cry:

**Pull request merged**